src/test/java/no/nav/tag/finnkandidatapi/kandidat/KandidatRepositoryTest.java

```diff
@@ -41,11 +41,35 @@ public void skal_kunne_lagre_og_hente_ut_med_null() {
         Kandidat behovTilLagring = enKandidatMedBareNull();
 
         Integer lagretId = repository.lagreKandidat(behovTilLagring);
-        Kandidat uthentetBehov = repository.hentKandidat(lagretId);
+        Kandidat uthentetBehov = repository.hentKandidat(lagretId).get();
 
         assertThat(uthentetBehov).isEqualToIgnoringGivenFields(enKandidatMedNullOgTommeSet(), "id");
     }
 
+    @Test
+    public void hentNyesteKandidat__skal_returnere_siste_registrerte_kandidat() {
+        Kandidat kandidat1 = enKandidat();
```

Comment:
Her er det ikke helt tydelig at kandidat1 og kandidat2 er forskjellige i utgangspunktet. Hva med å sette et konkret t.b. som er forskjellige på de to?